src/index.js

```diff
@@ -1,6 +1,30 @@
 import React, { Component } from 'react';
 import ReactDOM from 'react-dom';
-
+import { Provider, compose } from 'react-redux';
+import { createStore } from 'redux';
+import rootReducer from './reducers';
 import App from './components/App';
+import ReduxDevTools from './components/_dev/ReduxDevTools';
+
+/*
+const enhancer = compose(
+  ReduxDevTools.instrument()
+);
+*/
```

Comment:
@Spetastian What will this do?  (If you think it's a good reminder to have this here, we don't need to delete it.)